joomla/joomla-cms
# [5.3] Add Files folder to Media component and to "FileSystem local" adapter

Pull Request for Issue # .

### Summary of Changes

Trying to add `/files` folder, addittionaly to existing `/images` folder. 
It is not very logical to have PDF (and other) documents under `/images`. 

The changes affects new installations. Existing installations should continue to work as usual.


### Testing Instructions
Check that Media manager works.
Upload images/files in Media manager view, upload images using Media field in Article editing, etc.


### Actual result BEFORE applying this Pull Request
Works


### Expected result AFTER applying this Pull Request
Works


### Link to documentations
Please select:
- [x] Pull Request link for manual.joomla.org: https://github.com/joomla/Manual/pull/333
